app/frontend/wallet/constants.ts

```diff
@@ -15,88 +16,36 @@ export const TX_WITNESS_SIZES = {
   shelley: 139, //TODO: this is too much
   byronV1: 170,
 }
+
+export const enum NetworkId {
+  MAINNET = 1,
+  HASKELL_TESTNET = 0,
+}
+
+export const enum ProtocolMagic {
+  MAINNET = 764824073,
+  HASKELL_TESTNET = 42,
+}
+
 export const PROTOCOL_MAGIC_KEY = 2
-export const NETWORKS = {
-  BYRON: {
-    MAINNET: {
-      name: 'mainnet',
-      protocolMagic: 764824073,
-    },
-    TESTNET: {
-      name: 'testnet',
-      protocolMagic: 1097911063,
-    },
-  },
-  SHELLEY: {
-    MAINNET: {
-      name: 'mainnet',
-      networkId: 1,
-      protocolMagic: 764824073,
-      eraStartSlot: 4492800, // 21600 slot x 208 epochs
-      eraStartDateTime: Date.parse('29 Jul 2020 21:44:51 UTC'),
-      ttl: 3600,
-      epochsToRewardDistribution: 4,
-      minimalOutput: 1000000,
-    },
-    HASKELL_TESTNET: {
-      name: 'htn',
-      networkId: 0,
-      protocolMagic: 42,
-      ttl: 500,
-    },
-    INCENTIVIZED_TESTNET: {
-      name: 'itn',
-      addressDiscriminator: 'testnet',
-      // ./jcli rest v0 settings --output-format=json
-      chainConfig: {
-        block0Hash: '8e4d2a343f3dcf9330ad9035b3e8d168e6728904262f2c434a4f8f934ec7b676',
-        block0Time: '2019-12-13T19:13:37+00:00',
-        blockContentMaxSize: 1024000,
-        consensusVersion: 'genesis',
-        currSlotStartTime: '2019-12-22T10:31:36+00:00',
-        epochStabilityDepth: 10,
-        fees: {
-          certificate: 100000,
-          coefficient: 100000,
-          constant: 200000,
-          per_certificate_fees: {
-            certificate_pool_registration: 500000000,
-            certificate_stake_delegation: 400000,
-          },
-        },
-        rewardParams: {
-          compoundingRatio: {
-            denominator: 1,
-            numerator: 0,
-          },
-          compoundingType: 'Linear',
-          epochRate: 1,
-          epochStart: 1,
-          initialValue: 3835616440000,
-          poolParticipationCapping: [100, 100],
-          rewardDrawingLimitMax: {
-            ByStakeAbsolute: {
-              denominator: 10000000000,
-              numerator: 4109589,
-            },
-          },
-        },
-        slotDuration: 2,
-        slotsPerEpoch: 43200,
-        treasuryTax: {
-          fixed: 0,
-          ratio: {
-            denominator: 10,
-            numerator: 1,
-          },
-        },
-      },
-    },
+
+export const NETWORKS: {[key: string]: Network} = {
+  MAINNET: {
+    name: 'mainnet',
+    networkId: NetworkId.MAINNET,
+    protocolMagic: ProtocolMagic.MAINNET,
+    eraStartSlot: 4492800, // 21600 slot x 208 epochs
+    eraStartDateTime: Date.parse('29 Jul 2020 21:44:51 UTC'),
+    ttl: 3600,
```

Comment:
isn't that rather something like `defaultTtl`?

Reply:
I agree, I was thinking it doesnt have to be bound to a specific network, but then I thought, `ttlSeconds` might not be the best name since it just only happens that the current network has one slot per second but some other network may have another number of slots per second. Additionally the current `ttl` represents number of slot we add to the current slot number so if it makes sense to you I ll rename it to `ttlSlots`. Im not that happy with this name either so Im really open to suggestions. What do you think? :D

Reply:
Hm, but instead of defining the number of slots we add to the current slot we could define how many slots are per second (in this case 1) and have another constant for seconds and helper function that returns slotNumber appropriate for one hour from now. Anyway we can do this in a another PR.

Reply:
also here 
```
return cryptoProvider.network.eraStartSlot + timePassed + cryptoProvider.network.ttl
```
we are adding the number of passed seconds + ttl which would break if the `slot per second` ratio was different. We could fix that as well.